internal/state/change_processor_test.go

```diff
@@ -52,7 +52,7 @@ var _ = Describe("ChangeProcessor", func() {
 					},
 					Spec: v1alpha2.HTTPRouteSpec{
 						CommonRouteSpec: v1alpha2.CommonRouteSpec{
-							ParentRefs: []v1alpha2.ParentRef{
+							ParentRefs: []v1alpha2.ParentReference{
```

Comment:
I think we need to update this dependency to `v1beta1`: https://github.com/kubernetes-sigs/gateway-api/tree/v0.5.0/apis/v1beta1

Reply:
I didn't change it because I saw this in the repo
> The latest supported version is v1alpha2 as released by the [v0.4.3 release](https://github.com/kubernetes-sigs/gateway-api/releases/tag/v0.4.3) of this project. This version of the API is expected to graduate to beta in the future with relatively minimal changes.

and I thought v1beta1 wasn't supported for some reason, but I think they just forgot to update the README from reading the release notes.

I'll update the PR, thanks!

Reply:
this and all v1alpha2 across our code base :)

Reply:
updated it